plugin/collector/snap-collector-anothermock1/anothermock/anothermock.go

```diff
@@ -0,0 +1,145 @@
+/*
+http://www.apache.org/licenses/LICENSE-2.0.txt
+
+
+Copyright 2015 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package anothermock
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+	"time"
+
+	"github.com/intelsdi-x/snap/control/plugin"
+	"github.com/intelsdi-x/snap/control/plugin/cpolicy"
+	"github.com/intelsdi-x/snap/core"
+	"github.com/intelsdi-x/snap/core/ctypes"
+)
+
+const (
+	// Name of plugin
+	Name = "anothermock"
+	// Version of plugin
+	Version = 1
+	// Type of plugin
+	Type = plugin.CollectorPluginType
+)
+
+// Mock collector implementation used for testing
+type AnotherMock struct {
```

Comment:
It is needed in unit tests. There was already mock1 and mock2, but both of them have the same name mock, so framework just switches to new version instead of adding new metrics to running task (this was supported before). Another mock has another name and namespace, so task with /intel/*/foo metric will collect both /intel/mock/foo and /intel/anothermock/foo after this PR.